Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -56,10 +59,22 @@ struct GeometryEditorMenu: View {
         } label: {
             Label("Geometry Editor", systemImage: "pencil.tip.crop.circle")
         }
+        .task {
```

Comment:
Should these task modifiers be attached to the `editMenuContent`, or should they remains attached to the `Menu`? The former seems more correct semantically - you only needs updates when edit a geometry; but the latter seems more convenient - don't need to worry about the conditional view's lifecycle vs when the task modifiers are created, etc. 🤔